skin.json

```diff
@@ -214,27 +214,27 @@
     ]
   },
   "icons": {
-    "play": {"fontFamilyName": "alice", "fontString": "\u0068", "fontStyleClass": "oo-icon oo-icon-play"},
-    "pause": {"fontFamilyName": "alice", "fontString": "\u0067", "fontStyleClass": "oo-icon oo-icon-pause"},
-    "volume": {"fontFamilyName": "alice", "fontString": "\u0062", "fontStyleClass": "oo-icon oo-icon-volume-mobile"},
-    "volumeOff": {"fontFamilyName": "alice", "fontString": "\u0070", "fontStyleClass": "oo-icon oo-icon-volume-mobile-mute"},
-    "expand": {"fontFamilyName": "alice", "fontString": "\u0069", "fontStyleClass": "oo-icon oo-icon-resize-large"},
-    "compress": {"fontFamilyName": "alice", "fontString": "\u006a", "fontStyleClass": "oo-icon oo-icon-resize-small"},
-    "ellipsis": {"fontFamilyName": "alice", "fontString": "\u0066", "fontStyleClass": "oo-icon oo-icon-menu"},
-    "replay": {"fontFamilyName": "alice", "fontString": "\u0063", "fontStyleClass": "oo-icon oo-icon-upnext-replay"},
-    "share": {"fontFamilyName": "alice", "fontString": "\u006f", "fontStyleClass": "oo-icon oo-icon-topmenu-share"},
-    "cc": {"fontFamilyName": "alice", "fontString": "\u006B", "fontStyleClass": "oo-icon oo-icon-topmenu-cc"},
-    "discovery": {"fontFamilyName": "alice", "fontString": "\u006C", "fontStyleClass": "oo-icon oo-icon-topmenu-discovery"},
-    "quality": {"fontFamilyName": "alice", "fontString": "\u006D", "fontStyleClass": "oo-icon oo-icon-topmenu-quality"},
-    "setting": {"fontFamilyName": "alice", "fontString": "\u006E", "fontStyleClass": "oo-icon oo-icon-topmenu-settings"},
-    "dismiss": {"fontFamilyName": "alice", "fontString": "\u0065", "fontStyleClass": "oo-icon oo-icon-close"},
+    "play": {"fontFamilyName": "ooyala-slick-type", "fontString": "\u0068", "fontStyleClass": "oo-icon oo-icon-play-slick"},
```

Comment:
Do you guys use the font string at all? do you know if these are the same as before?